core/src/main/java/org/apache/iceberg/SnapshotSummary.java

```diff
@@ -35,6 +36,8 @@
   public static final String ADDED_RECORDS_PROP = "added-records";
   public static final String DELETED_RECORDS_PROP = "deleted-records";
```

Comment:
We still interpret this as the number of records in data files that were removed?